src/Bicep.Core.Samples/Files/Metadata_CRLF/main.bicep

```diff
@@ -0,0 +1,45 @@
+/* 
+  This is a block comment.
+*/
+
+// metadata with value
+meta myString2 = 'string value'

```

Comment:
You've got good coverage for the positive case, but will also need some negative tests (to ensure Bicep behaves as expected in various edge cases). Off the top of my head, some scenarios I think would be worth testing:
* 2 `meta` blocks with the same identifier name
* Invalid/blank identifier name
* Missing various pieces of the syntax (missing identifier, missing `=`, missing value)
* Ensure overlap with other symbols works correctly (see comment on permitting `meta` & `param` identifiers to overlap)
* `meta` blocks containing expressions (I assume we want to block this)
* Ensure that compilation doesn't crash if there are cycles (`meta foo = foo`)
* Decorators on `meta` are blocked with an error that makes sense